src/components/MultiHopTrade/MultiHopTrade.tsx

```diff
@@ -32,12 +41,14 @@ export const MultiHopTrade = (props: CardProps) => {
   const {
     state: { wallet },
   } = useWallet()
+  const [showTradeQuotes, handleToggleShowTradeQuotes] = useToggle(false)
```

Comment:
"Toggle" is already a verb, so we can probably drop the "handle" whilst still meeting convention.

```suggestion
  const [showTradeQuotes, toggleShowTradeQuotes] = useToggle(false)
```